src/ui/layout/layout.svelte

```diff
@@ -17,6 +17,8 @@
     height: 100%;
     container-type: size;
     container-name: layout-query-container;
+    overflow-y: auto;
+    overscroll-behavior: none;
```

Comment:
this prevents iOS from overscrolling and showing the contents of the page underneath when you overshoot